aiven/client/envdefault.py

```diff
@@ -6,15 +6,21 @@
 Configurable parameters via environment variables
 """
 
+from typing import Optional
+
 import os
 
-USER_HOME = os.path.expanduser("~")
+USER_HOME: str = os.path.expanduser("~")
 
-AIVEN_CONFIG_DIR = os.environ.get("AIVEN_CONFIG_DIR", os.path.join(USER_HOME, ".config", "aiven"))
+AIVEN_CONFIG_DIR: str = os.environ.get("AIVEN_CONFIG_DIR", os.path.join(USER_HOME, ".config", "aiven"))
 
-AIVEN_AUTH_TOKEN = os.environ.get("AIVEN_AUTH_TOKEN")
-AIVEN_CA_CERT = os.environ.get("AIVEN_CA_CERT")
-AIVEN_CLIENT_CONFIG = os.environ.get("AIVEN_CLIENT_CONFIG", os.path.join(AIVEN_CONFIG_DIR, "aiven-client.json"))
-AIVEN_CREDENTIALS_FILE = os.environ.get("AIVEN_CREDENTIALS_FILE", os.path.join(AIVEN_CONFIG_DIR, "aiven-credentials.json"))
-AIVEN_PROJECT = os.environ.get("AIVEN_PROJECT")
-AIVEN_WEB_URL = os.environ.get("AIVEN_WEB_URL")
+AIVEN_AUTH_TOKEN: Optional[str] = os.environ.get("AIVEN_AUTH_TOKEN")
+AIVEN_CA_CERT: Optional[str] = os.environ.get("AIVEN_CA_CERT")
+AIVEN_CLIENT_CONFIG: Optional[str] = os.environ.get(
```

Comment:
I think it should be just `str` - it has default value which can't be `None`.

In general I think `mypy` probably can infer most of the types in this module without explicit type specifications.